docs/sources/tempo/release-notes/v2-4.md

```diff
@@ -0,0 +1,248 @@
+* Added per-tenant compaction window. [PR 3129](https://github.com/grafana/tempo/pull/3129)
+* Added `--max-start-time` and `--min-start-time` flag to tempo-cli command `analyse blocks`. [PR 3250](https://github.com/grafana/tempo/pull/3250)
+* Added per-tenant configurable `remote_write` headers to metrics-generator. [#3175](https://github.com/grafana/tempo/pull/3175)
+* Added variable expansion support to overrides configuration. [PR 3175](https://github.com/grafana/tempo/pull/3175)
+* Added HTML pages `/status/overrides` and `/status/overrides/{tenant}`. [PR 3244](https://github.com/grafana/tempo/pull/3244) [#3332](https://github.com/grafana/tempo/pull/3332)
+* Precalculate and reuse the vParquet3 schema before opening blocks. [PR 3367](https://github.com/grafana/tempo/pull/3367)
+* Made the trace ID label name configurable for remote written exemplars. [PR 3074](https://github.com/grafana/tempo/pull/3074)
+* Performance improvements in span filtering by introducing `AttributePolicyMatch` & `IntrinsicPolicyMatch` structures to match span attributes based on strongly typed values & precompiled regexp. [PR 3025](https://github.com/grafana/tempo/pull/3025)
+* Merged the processors overrides set through runtime overrides and user-configurable overrides. [PR 3125](https://github.com/grafana/tempo/pull/3125)
+* Removed experimental websockets support for search streaming. GRPC is the supported method of streaming results. [PR 3307](https://github.com/grafana/tempo/pull/3307)
+* Introduced localblocks process configuration option to select only server spans. [PR 3303](https://github.com/grafana/tempo/pull/3303)
+* Localblocks processor honor tenant max trace size limit. [PR 3305](https://github.com/grafana/tempo/pull/3305)
+
+## Upgrade considerations
+
+When [upgrading]({{< relref "../setup/upgrade" >}}) to Tempo 2.4, be aware of these considerations and breaking changes.
+
+### Transition to vParquet 3
+
+vParquet3 format is now the default block format. It is production ready and we highly recommend switching to it for improved query performance and [dedicated attribute columns]({{< relref "../operations/dedicated_columns" >}}).
+
+Upgrading to Tempo 2.4 modifies the Parquet block format. Although you can use Tempo 2.3 with vParquet2 or vParquet3, you can only use Tempo 2.4 with vParquet3.
+
+With this release, vParquet is being deprecated. Tempo 2.4 will still read vParquet1 blocks. However, Tempo will fail to configure them. [[PR 3377](https://github.com/grafana/tempo/pull/3377/files#top)]
+
+For information on upgrading, refer to [Upgrade to Tempo 2.4](https://grafana.com/docs/tempo/latest/setup/upgrade/) and [Choose a different block format](https://grafana.com/docs/tempo/next/configuration/parque#choose-a-different-block-format) .
+
+### Updated, removed, or renamed configuration parameters
```

Comment:
These fields have all been removed in favor of the new cache config:

```
storage:
  trace:
    cache:
    search:
      cache_control:
    background_cache:
    memcached:
    redis:
```

maybe mention them here with a "see below for details"? up to you

Reply:
What do you think about adding the a new row to the table and including these with a link to the section below?